Source/Admin/Domain/CreateProjectValidator.cs

```diff
@@ -20,9 +20,9 @@ public CreateProjectValidator()
             RuleFor(_ => _.NationalSocietyId)
                 .NotEmpty()
                 .WithMessage("National society id is mandatory");
-            RuleFor(_ => _.SurveillanceContex)
-                .NotEmpty()
-                .WithMessage("Surveillance contex is mandatory");
+            //RuleFor(_ => _.SurveillanceContex)
```

Comment:
Could you please add a comment on why this rule is commented out? Should it be removed or is it WIP?